cookbooks/convection-box.prm

```diff
@@ -86,16 +86,15 @@ end
 # which boundaries actually carry a prescribed boundary temperature, whereas
 # all other parts of the boundary are insulated (i.e., no heat flux through
 # these boundaries; this is also often used to specify symmetry boundaries).
-subsection Model settings
+# The parameters below this comment were created by the update script
```

Comment:
Need to check that all comments in the cookbooks and benchmarks are preserved (fix in a later commit).